examples/post_training_quantization/torch/mobilenet_v2/main.py

```diff
@@ -0,0 +1,179 @@
+val_dataset = datasets.Food101(
+    root=DATASET_PATH,
+    split = 'test', 
+    transform = transforms.Compose([
+        transforms.Resize(256),
+        transforms.CenterCrop(224),
+        transforms.ToTensor(),
+        normalize,
+    ]),
+    download = True
+)
+val_loader = torch.utils.data.DataLoader(
+        val_dataset, batch_size=128, num_workers=4, shuffle=False)
+
+model = models.mobilenet_v2(num_classes=FOOD101_CLASSES) 
+model.eval()
+model = load_checkpoint(model)
+
+###########################################################################
+# Quantize a PyTorch model
+###########################################################################
+
+'''
+Transformation function transforms a data item into model input.
+
+To validate the transform function use the following code:
+>> for data_item in val_loader:
+>>    model(transform_fn(data_item))
+'''
+def transform_fn(data_item):
+    images, _ = data_item
+    return images
+
+calibration_dataset = nncf.Dataset(val_loader, transform_fn)
+quantized_model = nncf.quantize(model, calibration_dataset)
+
+###########################################################################
+# Benchmark performance, calculate compression rate and validate accuracy
+###########################################################################
+
+dummy_input = torch.randn(1, 3, 224, 224)
+ov_model = ov_convert(model.cpu(), dummy_input)
+ov_quantized_model = ov_convert(quantized_model.cpu(), dummy_input)
+
+print('[1/7] Benchmark FP32 model:')
+fp32_fps = ov_benchmark(ov_model, verbose=True)
+print('[2/7] Benchmark INT8 model:')
+int8_fps = ov_benchmark(ov_quantized_model, verbose=True)
+
+fp32_ir_path = f'{ROOT}/mobilenet_v2_fp32.xml'
+int8_ir_path = f'{ROOT}/mobilenet_v2_int8.xml'
+ov.serialize(ov_model, fp32_ir_path)
+ov.serialize(ov_quantized_model, int8_ir_path)
+
+fp32_model_size = ov_model_size(fp32_ir_path)
+int8_model_size = ov_model_size(int8_ir_path)
+print(f'[3/7] Save FP32 model: {fp32_ir_path} ({fp32_model_size:.3f} Mb)')
+print(f'[4/7] Save INT8 model: {int8_ir_path} ({int8_model_size:.3f} Mb)')
+
+# Using [the dynamic shape feature](https://docs.openvino.ai/latest/openvino_docs_OV_UG_DynamicShapes.html)
+# to compute the last batch of the validation dataset
+ov_model.reshape([-1, 3, 224, 224])
+ov_quantized_model.reshape([-1, 3, 224, 224])
+
+compiled_model = ov.compile_model(ov_model)
+compiled_quantized_model = ov.compile_model(ov_quantized_model)
+
+print('[5/7] Validate OpenVINO FP32 model:')
+fp32_top1 = validate(compiled_model, val_loader)
+print(f'Accuracy @ top1: {fp32_top1:.3f}')
+
+print('[6/7] Validate OpenVINO INT8 model:')
+int8_top1 = validate(compiled_quantized_model, val_loader)
+print(f'Accuracy @ top1: {int8_top1:.3f}')
+
+print('[7/7] Report:')
+print(f'Accuracy drop: {fp32_top1 - int8_top1:.3f}')
+print(f'Compression rate: {fp32_model_size / int8_model_size:.3f}')
+# https://docs.openvino.ai/2018_R5/_docs_IE_DG_Intro_to_Performance.html
```

Comment:
According to `examples/post_training_quantization/torch/mobilenet_v2/requirements.txt` `openvino-dev>=2022.2.0` is required, but the link refers to `OpenVINO 2018.5`. I think the information may be outdated or confuse users.

Reply:
Done